src/main/webapp/js/common.js

```diff
@@ -92,8 +92,9 @@ var StatusType = {
     INFO: 'info',
     WARNING: 'warning',
     DANGER: 'danger',
+    PRIMARY: 'primary',
     isValidType: function(type) {
-        return type === StatusType.SUCCESS || type === StatusType.INFO
+        return type === StatusType.SUCCESS || type === StatusType.INFO || type === StatusType.PRIMARY
```

Comment:
If you define a new status type here, define one similarly in `StatusMessageColor` enum in `Const` class.